src/app/components/home/home.component.ts

```diff
@@ -43,20 +43,19 @@ export class HomeComponent implements OnInit, OnDestroy {
 
   getMeetups() {
     this._scMeetup
-      .getAll()
+      .get(this.meetupsCurrentPage, this.meetupsPerPage)
       .pipe(takeUntil(this._destroySubscribes$))
       .subscribe((meetups) => {
-        this.meetupsAll = meetups.items;
-        this.hasMeetup = meetups.items.length > 0 ? true : false;
+        this.meetups.push(...meetups.items);
 
-        this.meetupsAll.sort((a, b) => (a.startDate > b.startDate ? -1 : 0));
-        this.meetups = this.meetupsAll.slice(0, 5);
+        const maxPage = Math.ceil(meetups.totalItems / meetups.itemsPerPage);
+        if (this.meetupsCurrentPage >= maxPage) this.showButtonMoreMeetups = false;
```

Comment:
@raffacabofrio, esse seu ir possui somente uma atribuição. Como uma condição retorna sempre true ou false, você pode fazê-la diretamente na atribuição. 

Alterei o sinal para menor, para não precisar fazer um not da condição anterior.

```suggestion
        this.showButtonMoreMeetups = this.meetupsCurrentPage < maxPage;
```